### ISSUE TITLE
Latest OMSagent for Linux installed - but VM is showing not connected in log analytics workspace. 

### ISSUE THREAD

**Issue by sumitsingh306** [microsoft/OMS-Agent-for-Linux#1332]
Hi Microsoft/Azure,

Hope I will get solution and guidance here regarding manual installation of OSM agent for linux on Azure VM. I checked the issue here in github and found #52  had similar issue what I'm facing now after 4 years. It seems issue not fix till date. 

I have installed OMSAgent_v1.13.33-0 on Azure Ubuntu VM using manual way. Agent got installed successfully and onboarded with Log analytics workspace. But when I go Log analytics workspace > Workspace Data Sources > Virtual Machines, my VM 'onsdag' is showing **not-connected.**  
As mentioned in issue no #52 , when we select the VM and click connect option from Azure portal, after 2-3 min it start showing connected. You can refer below image. Log are getting collected in workspace and VM name is showing when I run run query in log. 
I installed the agent from github Microsoft repository. 
Link >> wget https://github.com/microsoft/OMS-Agent-for-Linux/releases/download/OMSAgent_v1.13.33-0/omsagent-1.13.33-0.universal.x64.sh

![image](https://user-images.githubusercontent.com/45307895/116787052-3d039180-aaa2-11eb-863e-62070a692660.png)
![image](https://user-images.githubusercontent.com/45307895/116787164-ea76a500-aaa2-11eb-8bca-a2bb6b48bc26.png)
![image](https://user-images.githubusercontent.com/45307895/116787175-05e1b000-aaa3-11eb-8eaa-f75e3d9227c4.png)

Please clarify that issue is still in latest agent as well or it solved. If solved then why it not showing connected in workspace.

**Comment by sumitsingh306** [comment]
Hi @msftgits, can I expect any reply on my query.

**Comment by stupiddog1979** [comment]
I am experiencing the same problem with version 1.13.35. Manually connecting via Azure Portal website works, but is manual labour and very annoying.

**Comment by isinghrana** [comment]
I am experiencing the  same issue as well with one minor nuance. I am installing the OMS Agent on Azure Databricks using cluster init scripts, the VMs for Azure Databricks are in managed resource group on which we don't have any control so the Manually Connecting from Azure Portal does not work. I wasted quite a bit of time on this, fortunately I decided just to run queries on Perf table and found the data to be there. It would be good to have this documented somewhere.
